### DISCUSSION TITLE
Has anyone tried to publish the 0.3.0 connector to exchange ?

### DISCUSSION THREAD

**Discussion by Drashti065** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#68] [Q&A] [answered]
Hi Team 
I am trying to publish the connector to my org exchange and getting below error   : Failed to deploy artifacts: Could not transfer artifact 1ab8fee0-ac35-423b-ac3d-ZZZZZZZZZZZZZ:mulechain-ai-connector:pom:0.3.0 from/to Repository (https://maven.eu1.anypoint.mulesoft.com/api/v3/organizations/1ab8fee0-ac35-423b-ac3d-ZZZZZZZZZ/maven): status code: 412, reason phrase: Precondition Failed (412)
[issue_pom.txt](https://github.com/user-attachments/files/17210194/issue_pom.txt)

**Comment by mboss37** [comment, marked as answer]
Hi @Drashti065 

Please try doing the changes recommended by this article and let me know if it helped: https://help.salesforce.com/s/articleView?id=001120384&type=1

Best 
Mihael

**Reply by Drashti065** [reply]
Hello Mihael
thank you for pointer that was one change I made in the POM file & few other tweaks as well as added timeout in the settings.xml helped solve the issue and it is now published. 

Following are the changes I made : 
- maven 3.9.9 did not worked so moved back to 3.8.8
- added 2 properties in settings.xml - to resolve the timeout
-		<http.socket.timeout>120000</http.socket.timeout> 
         <http.connection.timeout>120000</http.connection.timeout> 

- added mulesoft-public repository in settings xml
- in the connector pom - I have to uncomment - org.jacoco plugin & dependencies.
-  I used command with -DskipTests to deploy and turned on -X for detailed logs to capture anything obvious. 
- https://help.salesforce.com/s/articleView?id=001120384&type=1  
- I have windows machine

**Comment by chiewklee** [comment]
Hi, 
Can you take a look at this doc - https://docs.mulesoft.com/exchange/to-publish-assets-maven and follow the steps in it?
Pl let us know if this resolve the issue.
Thanks.
